src/components/scoresheets/scoresheet_entry.jsx

```diff
@@ -1,192 +1,205 @@
 import React from "react";
 import PropTypes from "prop-types";
 import { connect } from "react-redux";
-import { createScoring, updateScoring } from '../../actions/scoringActions';
+import { createScoring, updateScoring } from "../../actions/scoringActions";
 import YouTube from "../video/YouTube";
 import "./scoresheet_entry.css";
 
 //=========================================
-// mapDispatchToProps
+// mapXToProps
 
-const mapDispatchToProps = (dispatch) => ({
-  createScoring: scoring => dispatch(createScoring(scoring)),
-  updateScoring: scoring => dispatch(updateScoring(scoring)),
+const mapStateToProps = (state, ownProps) => ({
```

Comment:
if mapStateToProps is not being used, let's remove it and pass null in as the first argument to connect